apps/charterafrica/app.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "charterafrica",
+  "scripts": {
+    "predeploy": "export NEXT_PUBLIC_APP_URL=$(dokku config:get charterafrica NEXT_PUBLIC_APP_URL) && export RESOURCES_SECRET_TOKEN=$(dokku config:get charterafrica RESOURCES_SECRET_TOKEN)",
+    "update-ecosystem-list": "curl -H \"x-api-key: $RESOURCES_SECRET_TOKEN\" $NEXT_PUBLIC_APP_URL/api/v1/resources/ecosystem/update-ecosystem-list",
+    "update-ecosystem-content": "curl -H \"x-api-key: $RESOURCES_SECRET_TOKEN\" $NEXT_PUBLIC_APP_URL/api/v1/resources/ecosystem/update-ecosystem-content"
+  },
```

Comment:
> 

1. This PR is only for Airtable integration. 
2. I didn't use `Airtable.js` just not to install an extra dependancy for something we can write.

Reply:
> Is `update-ecosystem-content` part of this PR? I'm assuming this will be done in GitHub integration.

I had this impleented to provide for a way to test this PR @kilemensi

Reply:
Cool, cool... I think airtable.js handles a few things:

1. Retry: https://airtable.com/developers/web/api/rate-limits
2. Parameter encoding: https://airtable.com/developers/web/api/list-records

If we're sure we can manage, all good.

Reply:
We can manage parameter encoding since we are using tableID instead of tableName but Retry could be needed if the list grows

Reply:
> Is update-ecosystem-content part of this PR? I'm assuming this will be done in GitHub integration.
> 
> I had this impleented to provide for a way to test this PR @kilemensi

I was talking about updating content. This PR is all about updating list, no?

Reply:
Update content.
- Content source is airtable so update content implies getting data from airtable.

Update List.
- Pull data from github and update locally existing list

Reply:
**List** is from airtable i.e. which contributors, organisations and tools we need to track.
**Content** is the actual details of an entity that we need to pull from sources such as GitHub. Content depends on List.